test/verify/check-superuser

```diff
@@ -134,11 +135,15 @@ session include system-auth
     @nondestructive
     def testWrongPasswd(self):
         b = self.browser
+        m = self.machine
 
         # Log in with limited access
         self.login_and_go(superuser=False)
         b.check_superuser_indicator("Limited access")
 
+        if m.image == "arch":
+            self.addCleanup(m.execute, "rm -f /var/run/faillock/admin")
```

Comment:
This should be in the previous commit. If you squash them, that'll go into the right place.